server/app/services/program/ProgramServiceImpl.java

```diff
@@ -321,6 +328,24 @@ private boolean isValidAbsoluteLink(String url) {
     return url.isBlank() || url.startsWith("http://") || url.startsWith("https://");
   }
 
+  /**
+   * When an admin deletes a custom confirmation screen, we want to also clear out all associated
+   * translations
+   */
+  private LocalizedStrings maybeClearConfirmationScreenTranslations(
+      ProgramDefinition programDefinition, Locale locale, String confirmationScreen) {
```

Comment:
**nitpick (blocking):** "screen" is an existing concept in the system synonymous with "block". Recommend referring to the string value of the custom confirmation message as `confirmationMessage` (maybe?) in the doc, variables, and method name here to avoid confusion